src/modules/evaluateExpression/evaluateExpression.ts

```diff
@@ -1,45 +1,9 @@
-interface IParameters {
-  [key: string]: any
-  connection?: IConnection
-}
-
-interface IQueryNode {
-  value?: string | number | boolean | object
-  type?: NodeType
-  operator?: Operator
-  children?: Array<IQueryNode>
-}
-
-interface IConnection {
-  user?: string
-  host?: string
-  database?: string
-  password?: string
-  port?: number
-  query?: any
-}
-
-type NodeType = 'string' | 'number' | 'boolean' | 'array'
-
-type Operator =
-  | 'AND'
-  | 'OR'
-  | 'CONCAT'
-  | '='
-  | '!= '
-  | '+'
-  | 'REGEX'
-  | 'objectProperties'
-  | 'pgSQL'
-  | 'graphQL'
-
-const defaultParameters: IParameters = {
-  connection: {},
-}
+import { IQueryNode, IParameters } from '../../types'
+import PostgresDB from '../../components/postgresConnect'
```

Comment:
I think having the Connection object passed in as a parameter is better for this module. Because if its being used without PostGres, there's no need to import or install any packages. For example, when the front-end uses this module, we don't want to have to have a post-gres package installed as we'll never use it. So the connection should be handled outside this module and just passed in if/when required.

Reply:
Ah, that's a good point. Sorry I wasn't seen that. : ) Will revert this right away.